cli/register/register.go

```diff
@@ -232,10 +235,18 @@ func DoIt(org, pattern, nodeIdTok, userPw, inputFile string, nodeOrgFromFlag str
 			if nodeType != n.GetNodeType() {
 				cliutils.Fatal(cliutils.CLI_INPUT_ERROR, msgPrinter.Sprintf("Node type mismatch. The node type '%v' does not match the node type '%v' of the Exchange node %v.", nodeType, n.GetNodeType(), nId))
 			}
+			existingNodeName = n.Name
 			break
 		}
 	}
 
+	if existingNodeName != "" && existingNodeName != nodeName {
+		msgPrinter.Printf("Updating node name...")
+		msgPrinter.Println()
+		patchNodeReq := cliexchange.NodeExchangePatchName{Name: nodeName}
+		cliutils.ExchangePutPost("Exchange", http.MethodPatch, cliutils.GetExchangeUrl(), "orgs/"+org+"/nodes/"+nodeId, cliutils.OrgAndCreds(userOrg, userAuth), []int{201}, patchNodeReq, nil)
```

Comment:
By now the node id and token is known for all cases, please use node id and token as cred for this call. You need to save the node id and the token for different cases so that they can be used here.

Reply:
Thanks. Updated creds to use nodeIdTok. Doesn't seem like it needed to be re-saved for different cases - if the token is regenerated - the nodeIdTok value will be updated